### DISCUSSION TITLE
New version of Librewolf unavailable on Homebrew Formulae [Updated]

### DISCUSSION THREAD

**Discussion by cristobalo7** [Homebrew/discussions#5634] [Casks] [answered]
### Output of `brew config`

```shell
irrelevant
```

### Output of `brew doctor`

```shell
irrelevant
```

### Description of issue

Hi,

The new version of Librewolf 130.0.1-1 is not available on Homebrew Cask.

https://formulae.brew.sh/cask/librewolf

The latest version on Homebrew Formulae is still 130.0,3.

Thank you.

### Relevant casks

https://formulae.brew.sh/cask/librewolf

**Comment by daeho-ro** [comment, marked as answer]
- https://github.com/Homebrew/homebrew-cask/pull/186557

**Reply by cristobalo7** [reply]
new version received, thank you.
